WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,178 @@
+
+# Index
+- Pre-Install
+- Install
+- Pre-Mix
+- Mixing
+- Post-Mix
+- Meta
+- Acknowledgements
+
+# Pre-Install
+
+### Who is behind Wasabi?
+
+The Company that is developing Wasabi is zkSNACKs LTD ([twitter](@Zksnacks_LTD) | [website](https://zksnacks.com/))  
+Members as of 2019.05.08:
+
+| Name              | Twitter                                                 | Position			   |
+| ------------------| --------------------------------------------------------|	-----------------------|
+| Gergely Hajdú 	| [@gergely_hajdu](https://twitter.com/gergely_hajdu) 	  | Co-CEO & Co-Founder	   |
+| Bálint Harmat		| [@bharmat84](https://twitter.com/bharmat84) 			  | Co-CEO & Co-Founder    |
+| Ádám Ficsór		| [@nopara73](https://twitter.com/nopara73)	              | Developer & Co-Founder |
+| Lucas Ontivero	| [@lontivero](https://twitter.com/)                      | Developer & Co-Founder |
+| Dan Walmsley   	| ([@danwalmsley](https://github.com/danwalmsley))        | Developer              |
+| Dávid Molnár   	| ([@molnardavid84](https://twitter.com/molnardavid84))   | Developer              |
+
+# Install
+
+### How do I install Wasabi?
+
+Follow [this guide](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/Guides/InstallInstructions.md)
+
+If you use Windows there is a [video guide](https://youtu.be/D8U53PFEsVk).
+
+### Do I need to run Tor?
+
+All Wasabi network traffic goes via Tor by default - no need to set up Tor yourself. If you do already have Tor, and it is running, then Wasabi will try to use that first.  
+
+# Pre-Mix
+
+### My wallet can't send to Bech32 addresses - what wallets can I use instead? 
+
+Wasabi generates Bech32 addresses only. These addresses start with the characters bc1... Some wallets/exchanges don't yet support this type of address and my give an error message (e.g. "unknown bitcoin address"). The solution is to manage your funds with a wallet which does support Bech32. Of [all the wallets](https://en.bitcoin.it/wiki/Bech32_adoption) that support Bech32, only the following wallets have Coin Control;
+
+| Platform 						| Recommended 																					| Other										|
+| ---------						| -----------------------------------------------------	|------------------------	|
+| Mobile (Android)		| [Samourai](https://samouraiwallet.com/)								| [Electrum](https://play.google.com/store/apps/details?id=org.electrum.electrum)								|
+| Desktop 						| [Bitcoin Core](https://bitcoincore.org/en/download/)	| [Electrum](https://electrum.org)	|
+
+I am not aware of any wallets for iOS which support coin control. If you are aware of any other Wallets with coin control please make a pull request.
+
+Be careful. If you send all your coins from an old wallet to a new wallet (from the table above) in one transaction then you will merge all your coins which is bad for privacy. Instead, **import your seed into one of the wallets in the table.**
```

Comment:
This is, while good, an impractical advice unfortunately :(

Reply:
How so?

Reply:
1. 99% of users would fail to figure out what to do.
2. Gap limit issues.
3. Core is not compatible with other wallets.
4. Wasabi is only looking at bech32 keys, this can result in a lot of confusion.
5. Keypath issues.
6. xpub format issues: xpub hex, xpub, ypub, zpub, etc are not standards and used differently wallet by wallet.
7. Import Wasabi seed to a mobile wallet and you doxxed yourself and all your past mixes to the company.

And these are just from the top of my head.